p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -3363,6 +3363,39 @@ manipulate the elements at the front and back of the stack:
   be a positive integer that is a compile-time known value. The return
   type is ```void```.
 
+The pseudocode below defines the behavior of ```push_front``` and ```pop_front```.
+
+~ Begin P4Pseudo
+void push_front(int count) {
```

Comment:
OK, given your later comment, I am not planning on changing this.  Github still thinks there are changes requested, though.  Not sure how to change that.